### DISCUSSION TITLE
Blank Panel in few area

### DISCUSSION THREAD

**Discussion by shetubd** [coder/code-server#3561] [Q&A] [answered]
I use code server in docker. It is show blank panel in few extension settings panel. 
![Untitled](https://user-images.githubusercontent.com/37763269/121076565-611c6600-c7f8-11eb-9b47-8461ce5d2dca.jpg)

code-server: v3.10.2
VS Code: v1.56.1
Commit: 387b12ef4ca404ffd39d84834e1f0776e9e3c005
Date: 2021-05-21T23:13:00Z
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0

**Comment by jsjoeio** [comment, marked as answer]
This is really strange 🤔 I am not super familiar with the extension you're using (SQL Tools?). Could you open a bug report and post reproduction steps?

**Reply by shetubd** [reply]
This is not only SQL Tools, I tried formulahendry.vscode-mysql tools. I see blank page in Extension details panel too.
I do not understand that is it my problem in code-server settings.
![Prettier](https://user-images.githubusercontent.com/37763269/121116906-18d16800-c839-11eb-95aa-89c9275df5d5.png)

Here is docker-compose file.
`# /var/lib/docker/volumes/nfs_www/_data/.docker/vs-code
services:
version: "2.1"
services:
  code-server:
    image: linuxserver/code-server
    container_name: t_code
    environment:
      - PUID=1001
      - PGID=1002
      - TZ=Asia/Dacca
      - PASSWORD=123456 #optional
      - HASHED_PASSWORD= #optional
      - SUDO_PASSWORD=123456 #optional
      - SUDO_PASSWORD_HASH= #optional
      - PROXY_DOMAIN=code-server.my.domain #optional
    volumes:
      - ./config:/config
      - nfs_www:/config/workspace
      - "/usr/bin/docker:/usr/bin/docker"     
      - "/var/run/docker.sock:/var/run/docker.sock"
      - "/usr/local/bin/docker-compose:/usr/local/bin/docker-compose"
    ports:
      - 8443:8443
      - 5500:5500
    restart: unless-stopped
# Docker Networks
networks:
    default:
        external:
           name: pihole
#Volumes
volumes:
    nfs_www:
      external: true
`

**Reply by jsjoeio** [reply]
It could be code-server or the extension or your settings but it's hard to tell. Sorry, I should have clarified. Please open a bug report [here](https://github.com/cdr/code-server/issues/new?assignees=&labels=&template=bug-report.md&title=) and then we can take a closer look!
